app/(gcforms)/[locale]/(form administration)/form-builder/Start.tsx

```diff
@@ -42,21 +43,21 @@ export const Start = () => {
       fileReader.onload = (e) => {
         if (!e.target || !e.target.result || typeof e.target.result !== "string") return;
         const data = safeJSONParse(e.target.result, cleaner);
-        if (data?.error) {
+        if (!data) {
           setErrors([{ message: t("startErrorParse") }]);
           target.value = "";
           return;
         }
 
-        const validationResult = validateTemplate(data);
+        const validationResult = validateTemplate(data as FormRecord);
 
         if (!validationResult.valid) {
           setErrors(validationResult.errors);
           target.value = "";
           return;
         }
 
-        importTemplate(data);
+        importTemplate(data as FormProperties);
```

Comment:
This seems to be wrongly typed and should be: 
```suggestion
        importTemplate(data.form);
```